File: app/src/main/java/com/bas/google_book_app/BookApp.java
package com.bas.google_book_app;

import android.app.Application;

import timber.log.Timber;

public class BookApp extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        startTimber();
    }

    private void startTimber() {
        Timber.plant(new Timber.DebugTree());
    }
}

File: app/src/main/java/com/bas/google_book_app/BookApp.kt
package com.bas.google_book_app

import android.app.Application
import timber.log.Timber
import timber.log.Timber.DebugTree

class BookApp : Application() {
    override fun onCreate() {
        super.onCreate()
        startTimber()
    }

    private fun startTimber() {
        Timber.plant(DebugTree())
    }
}